tests/models/test_attention_processor.py

```diff
@@ -90,9 +90,7 @@ def test_conversion_when_using_device_map(self):
         ).images
 
         # the initial conversion succeeds
-        pipe = DiffusionPipeline.from_pretrained(
-            "hf-internal-testing/tiny-stable-diffusion-pipe", device_map="sequential", safety_checker=None
```

Comment:
I am pretty sure the objective here was not to test for the "sequential" strategy.